src/domain/mongoose_domain_sql.erl

```diff
@@ -80,6 +84,18 @@ start(#{db_pool := Pool}) ->
                    "domain_settings.enabled = ", True/binary, ") "
               " WHERE domain_events.id >= ? AND domain_events.id <= ? "
               " ORDER BY domain_events.id ">>),
+    %% Admins
+    prepare(domain_insert_admin, domain_admin, [domain, password],
+            <<"INSERT INTO domain_admin (domain, password) VALUES (?, ?)">>),
+    prepare(domain_update_admin, domain_admin, [password, domain],
```

Comment:
I think the order was good here because the first `?` occurred is for the password and the second one is for the domain. The arguments passed during the execution were reversed indeed. I fixed it and added tests.